src/Pmi/Controller/ReviewController.php

```diff
@@ -29,12 +31,15 @@ class ReviewController extends AbstractController
 
     protected function getTodayRows($db, $today, $site)
     {
-        $ordersQuery = 'SELECT participant_id, \'order\' as type, id, order_id, created_ts, collected_ts, processed_ts, finalized_ts, finalized_samples, ' .
-            'greatest(coalesce(created_ts, 0), coalesce(collected_ts, 0), coalesce(processed_ts, 0), coalesce(finalized_ts, 0)) AS latest_ts ' .
-            'FROM orders WHERE ' .
-            '(created_ts >= :today OR collected_ts >= :today OR processed_ts >= :today OR finalized_ts >= :today) ' .
-            'AND (site = :site OR collected_site = :site OR processed_site = :site OR finalized_site = :site) ';
-        $measurementsQuery = 'SELECT participant_id, \'measurement\' as type, id, null, created_ts, null, null, finalized_ts, null, coalesce(finalized_ts, created_ts) as latest_ts ' .
+        $ordersQuery = 'SELECT o.participant_id, \'order\' as type, o.id, o.order_id, o.created_ts, o.collected_ts, o.processed_ts, o.finalized_ts, o.finalized_samples, ' .
+            'greatest(coalesce(o.created_ts, 0), coalesce(o.collected_ts, 0), coalesce(o.processed_ts, 0), coalesce(o.finalized_ts, 0), coalesce(oh.created_ts, 0)) AS latest_ts, ' .
+            'oh.type as oh_type ' .
+            'FROM orders o ' .
+            'INNER JOIN orders_history oh ' .
```

Comment:
Shouldn't this be a left join?  Otherwise, any orders without a history item will not show up in the today view.